x-pack/plugins/lists/public/exceptions/components/builder/entry_renderer.tsx

```diff
@@ -194,7 +196,9 @@ export const BuilderEntryItem: React.FC<EntryItemProps> = ({
   );
 
   const renderOperatorInput = (isFirst: boolean): JSX.Element => {
-    const operatorOptions = onlyShowListOperators
+    const operatorOptions = operatorsList
```

Comment:
nitpick: Better as `!!operatorsList` since `operatorsList` is an array.

Reply:
There is an es-lint rule that gives me an error with this:
Redundant double negation.eslintno-extra-boolean-cast

Reply:
Ah yes. 👍